src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -214,12 +223,16 @@ public void feederShoot() {
 	}
 
 	// sensor methods
-	public boolean getIndexSensor() {
-		return indexSensor.get();
+	public boolean indexSensorHasNote() {
+		return !indexSensor.get() && !getSensorOverride();
```

Comment:
Why is `indexSensor.get()` inverted now?

Reply:
.get() returns true as a default, and if it senses something it returns false